### ISSUE TITLE
Stack trace from go 1.13 style errors?

### ISSUE THREAD

**Issue by shouichi** [getsentry/sentry-go#373]
## Summary

It would be nice if the SDK extracts stack trace from go 1.13 style errors. For example,

```
err := errors.New("root cause")
err = fmt.Errorf("wrapping: %w", err)
```

## Motivation

From go 1.13, the `errors` package introduced wrapping/unwrapping errors. It would be a standard way to handle errors.

## Additional Context

It seems that the SDK currently supports

- golang.org/x/xerrors
- github.com/pkg/errors
- github.com/pingcap/errors
- github.com/go-errors/errors

and it successfully extracts stack trace from errors.

**Comment by rhcarvalho** [comment]
Hi! The standard library `errors` package is already supported, and the SDK knows how to unwrap and report error chains.

https://github.com/getsentry/sentry-go/blob/8f1d2c2ea0097440401a94aaa7be04e09b74143a/client.go#L469-L470

Support was introduced by https://github.com/getsentry/sentry-go/pull/206 and released in v0.6.0.

**Comment by shouichi** [comment]
Confirmed that SDK (v0.11.0) supports go 1.13 style errors. Turned out I misinterpreted the sentry's web UI. Thanks for the clarification 😄
